examples/adhesive_simulation_rough.ipynb

```diff
@@ -0,0 +1,870 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "4d8cfae0-2bd6-431c-9acd-cc1a4e5490a4",
+   "metadata": {},
+   "source": [
+    "# Example of rough simulation with adhesion on the contact challenge 512x512 topography\n",
+    "\n",
+    "The adhesion is longer ranged in this example than in the original contact challenge problem definitition because we use a much coarser grid here. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 182,
+   "id": "0368561e-3562-44bc-8586-192e8e424b22",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from SurfaceTopography import read_published_container\n",
+    "import Adhesion # required to register the \"make_system\" function\n",
+    "from Adhesion.Interactions import Exponential\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import matplotlib as mpl\n",
+    "\n",
+    "from matplotlib.colors import LinearSegmentedColormap"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "53167755-a8ad-4fe9-924f-d7ae3e260a1d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "c, = read_published_container(\"https://doi.org/10.57703/ce-vrcd9\")"
```

Comment:
Change to : 
 "c, = read_published_container(\"https://doi.org/10.57703/ce-rs7jq")"

Reply:
can you actually edit the notebook ? 

and remove the workarounds I did to make the surface periodic?